### PR TITLE
NEPT-1125: Added function to set up main menu

### PR DESCRIPTION
**Pull request by imanoleguskiza**
### Description

Implement the horizontal menu in Europa theme

### Change log

- Added: Main menu positioning in Europa

### Commands

No commands

### CODE REVIEW

**Comment by voidtek** [comment]
:+1:

**Comment by drupol** [comment]
👍

File: profiles/common/modules/custom/nexteuropa_ecl_migrate/nexteuropa_ecl_migrate.module [hunk 1]
```diff
@@ -180,3 +181,10 @@ function _nexteuropa_ecl_migrate_dt_menus() {
 
   module_invoke('context', 'save', $context);
 }
+
+/**
+ * Custom function that moves the proper blocks in the right areas.
+ */
+function _nexteuropa_ecl_migrate_europa_main_menu() {
+  multisite_drupal_toolbox_add_block_context('site_wide', 'main-menu', 'system', 'main-menu', 'navigation');
```
Comment (Fefaine) [inline profiles/common/modules/custom/nexteuropa_ecl_migrate/nexteuropa_ecl_migrate.module]: The region 'navigation' does not exist in Europa, i don't see that region on the info file

Comment (voidtek) [inline profiles/common/modules/custom/nexteuropa_ecl_migrate/nexteuropa_ecl_migrate.module]: In the europa.info, the first one is navigation

; ========================================
; Regions
; ========================================
regions[navigation] = Navigation

Comment (Fefaine) [inline profiles/common/modules/custom/nexteuropa_ecl_migrate/nexteuropa_ecl_migrate.module]: 👓

Comment (drupol) [inline profiles/common/modules/custom/nexteuropa_ecl_migrate/nexteuropa_ecl_migrate.module]: 🤙

**Pull request merged**